File: src/SextantTG/SextantTG.ActiveRecord/PictureComment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace SextantTG.ActiveRecord
{
    /// <summary>
    /// 图片评论实体类
    /// </summary>
    public class PictureComment : BaseAR
    {
        /// <summary>
        /// 评论ID
        /// </summary>
        public string CommentId { get; set; }

        /// <summary>
        /// 评论人
        /// </summary>
        public string CommentUserId { get; set; }

        /// <summary>
        /// 创建时间
        /// </summary>
        public DateTime? CreatingTime { get; set; }

        /// <summary>
        /// 图片ID
        /// </summary>
        public string PictureId { get; set; }

        /// <summary>
        /// 评论
        /// </summary>
        public string Comment { get; set; }
    }
}

File: src/App/src/com/ss/stg/model/PictureComment.java
package com.ss.stg.model;

import java.io.Serializable;
import java.util.Date;

/**
 * 图片评论
 * 
 * @author ggxx
 * 
 */
public class PictureComment implements Serializable {

	/**
	 * 
	 */
	private static final long serialVersionUID = -1673424426235035335L;
	private String commentId;
	private String commentUserId;
	private Date creatingTime;
	private String pictureId;
	private String comment;

	public PictureComment() {
	}

	public String getCommentId() {
		return commentId;
	}

	public void setCommentId(String commentId) {
		this.commentId = commentId;
	}

	public String getCommentUserId() {
		return commentUserId;
	}

	public void setCommentUserId(String commentUserId) {
		this.commentUserId = commentUserId;
	}

	public Date getCreatingTime() {
		return creatingTime;
	}

	public void setCreatingTime(Date creatingTime) {
		this.creatingTime = creatingTime;
	}

	public String getPictureId() {
		return pictureId;
	}

	public void setPictureId(String pictureId) {
		this.pictureId = pictureId;
	}

	public String getComment() {
		return comment;
	}

	public void setComment(String comment) {
		this.comment = comment;
	}

}
